_integration-schemas/codat/v1/items.md

```diff
@@ -0,0 +1,94 @@
+---
+tap: "codat"
+version: "1"
+key: "item"
+
+name: "items"
+doc-link: "https://docs.codat.io/reference/items"
+singer-schema: "https://github.com/singer-io/tap-codat/blob/master/tap_codat/schemas/items.json"
+description: |
+  The {{ table.name }} table contains information about items for a given company in your {{ integration.display_name }} account.
+
+replication-method: "Key-based Incremental"
+replication-key: "modifiedDate"
```

Comment:
It looks like `modifiedAt` is actually an attribute in the table, so this isn't needed